pkg/scheduler/framework/interface.go

```diff
@@ -105,6 +106,28 @@ const (
 	MaxTotalScore int64 = math.MaxInt64
 )
 
+// PodsToMoveKey is a reserved state key for stashing pods that
+// will be moved to activeQ at the end of a scheduling and binding cycle,
```

Comment:
this is really depending on when the pods are added.

Reply:
comments polished.